dashboard/db/migrate/20160517173415_add_users_to_unit_and_module_assignments.rb

```diff
@@ -0,0 +1,16 @@
+class AddUsersToUnitAndModuleAssignments < ActiveRecord::Migration
+  def change
+    add_reference :plc_enrollment_unit_assignments, :user, index: true
+    add_reference :plc_enrollment_module_assignments, :user, index: true
+
+    Plc::EnrollmentUnitAssignment.all.each do |unit_assignment|
+      unit_assignment.update(user: unit_assignment.plc_user_course_enrollment.user)
```

Comment:
All of these are rollback-able I believe, but I'll try it out once I'm out of the DSL branch. If not, I'll make this an up with a down migration to drop the above two columns.